programs/ssl/ssl_context_info.c

```diff
@@ -379,7 +379,7 @@ size_t read_next_b64_code( uint8_t **b64, size_t *max_len )
     int valid_balance = 0;  /* balance between valid and invalid characters */
     size_t len = 0;
     char pad = 0;
-    char c = 0;
+    signed char c = 0;
```

Comment:
This change doesn't look right. `c` contains a character read from a file, so `char` is the correct type.

Reply:
> This change doesn't look right. `c` contains either a character read from a file or EOF, so the correct type is `int`.
> 
> This bug fix should have a non-regression test, with an input file containing the byte `'\xff'`.

Thanks for the review.

When you say it should have "non-regression test, with an input file... ", where are you expecting this test ? The bug fix is for one of the program file. And it seems that takes command line args for testing different cases.  I have not explored ssl code in mbedtls yet, just started looking at this file because of failure with cmake to build it, so I might be missing something.

Reply:
There are tests for this program in `tests/context-info.sh`